packages/modul-components/src/components/menu/menu.ts

```diff
@@ -129,15 +131,19 @@ export class MMenu extends BaseMenu implements Menu {
             this.internalItems.forEach((item) => {
                 if (!item.isDisabled) {
                     if (item.group) {
-                        if (!this.open) {
-                            item.propOpen = false;
-                        }
                         let groupSelected: boolean = false;
                         item.$children.forEach(itemGroup => {
-                            if (itemGroup instanceof MMenuItem && itemGroup.selected) {
-                                item.propOpen = true;
-                                groupSelected = true;
-                            }
+                            itemGroup.$children.forEach(subItem => {
+                                if (subItem instanceof MMenuItem) {
+                                    subItem.selected = MMenuItemHelper.isRouterLinkActive(subItem);
+                                }
+
+                                if (subItem instanceof MMenuItem && subItem.selected) {
```

Comment:
Enlever ce if et dans ton if plus haut
item.propOpen = this.selected;
item.selected = this.selected;
groupSelected = this.selected;

Reply:
ifaut faire au min une if comme ça
![image](https://user-images.githubusercontent.com/53451444/64352677-74b6ef00-cfca-11e9-85c7-7c2776ff8e88.png)

sinon on va avoir cet erreur

![image](https://user-images.githubusercontent.com/53451444/64352653-6963c380-cfca-11e9-9031-5c0c27443069.png)